libsolidity/analysis/StaticAnalyzer.cpp

```diff
@@ -346,24 +346,19 @@ bigint StaticAnalyzer::structureSizeEstimate(Type const& _type, set<StructDefini
 	case Type::Category::Array:
```

Comment:
I think this function may be missing one more case: external function pointers.
Are such pointers allowed in structs? Because e.g. `function(uint) external pure returns (uint)` requires two stack slots.

Reply:
I'm making a note to ask @ekpyron.

Reply:
Function types, even external ones, require only one slot in storage.

Reply:
Oh, right. They're packed into 24 bytes in storage.

I remembered that they take up 2 slots but this must be just on stack then...
https://github.com/ethereum/solidity/blob/develop/libsolidity/ast/Types.cpp#L3092-L3098